packages/mdc-button/_mixins.scss

```diff
@@ -63,8 +64,8 @@
   // Note: Adjust padding to maintain consistent width with non-stroked buttons
   $padding-value: max($mdc-button-horizontal-padding - $stroke-width, 0);
 
-  padding-right: $padding-value;
-  padding-left: $padding-value;
+  @include mdc-rtl-reflexive-property(padding, $padding-value, $padding-value);
```

Comment:
Why are we doing this? This ends up adding more CSS for no gain in cases where both left and right have the same value.